ui/app/AppLayouts/Profile/Sections/Ens/TandC.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.14
```

Comment:
The name of this component is very confusing. I though it said "Tand C" lol
Maybe it would be better as `TermsAndC`